overlord/hookstate/hooktest/handler.go

```diff
@@ -32,6 +32,13 @@ type MockHandler struct {
 	ErrorCalled bool
 	ErrorError  bool
 	Err         error
+
+	SetConfCalls []confParameters
```

Comment:
Similarly, we don't want this type to become a massive union of everything all handlers want to do, but this will probably be accommodated naturally once we move SetConf out of Handler.